src/app/shared/runtime-console/space-namespace.service.ts

```diff
@@ -126,6 +128,10 @@ export class SpaceNamespaceService {
   }
 
   buildNamespace(): Observable<string> {
+    return this.devNamespace.namespace;
+/*
+    // TODO I don't grok why we were prepending user names in front of namespaces - kinda odd!
```

Comment:
This TODO should be converted into an actionable statement (maybe something less personal like TODO - review user names being prepended as the use case is not clear)